index.html

```diff
@@ -1,17 +1,11 @@
-<html>
-    <head>
-        <title>
-         tank     
-         
-         <p>
-             how are you?
-         </p>
-        </title>
-    </head>
-
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Tank</title>
+  </head>
+  <body>
     hello world!
```

Comment:
I can see you have not put a tag for your text "Hello world!" 
Try to use semantic HTML tags for your code to be professional, for that I suggest you use h1 tag
so your code will look like `<h1>hello world!</h1>`